config/application.rb

```diff
@@ -24,16 +23,18 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
 
+    # Please, add to the `ignore` list any other `lib` subdirectories that do
+    # not contain `.rb` files, or that should not be reloaded or eager loaded.
+    # Common ones are `templates`, `generators`, or `middleware`, for example.
+    config.autoload_lib(ignore: %w[assets tasks])
+
     # Configuration for the application, engines, and railties goes here.
     #
     # These settings can be overridden in specific environments using the files
     # in config/environments, which are processed later.
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
-
-    # Set asset path to be application specific so that we can put all GOV.UK
-    # assets into an S3 bucket and distinguish app by path.
-    config.assets.prefix = "/assets/short-url-manager"
```

Comment:
It's probs not an issue but this jumped out at me a bit. I wonder if we want to retain it? Not 100% sure if it's an issue though tbh

Reply:
It looks like other apps that have been upgraded had retained this bit of config so i suspect we probs should leave it in